tests/unit/models/MessageTest.php

```diff
@@ -1,61 +0,0 @@
-<?php namespace RainLab\Translate\Tests\Unit\Models;
```

Comment:
The test should be rewritten not removed. Has @daftspunk provided input on this change?

Reply:
No input on the change yet. I'll adapt the tests